connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/testutil/command/PublishMessageCommandDummy.java

```diff
@@ -41,6 +41,11 @@ public PublishMessageCommandStep3 correlationKey(String correlationKey) {
     return this;
   }
 
+  @Override
+  public PublishMessageCommandStep3 withoutCorrelationKey() {
+    return null;
```

Comment:
shouldn't this be `return this;` for consistency ?

Reply:
Lol good catch